src/Commands/Parameters/StringLiteralParameter.ts

```diff
@@ -0,0 +1,23 @@
+import { Parameter } from "./Parameter";
+
+/**
+ * A string literal parameter.
+ */
+export class StringLiteralParameter extends Parameter {
+    /**
+     * The name of the string literal.
+     */
+    public name: string;
+
+    /**
+     * Initializes a new instance of the Parameter class.
+     * 
+     * @param descriptor   A plain-text description of the parameter.
```

Comment:
After looking over the provided docs from the beginning of the semester, I don't know what you're referring to. Do I change the comment explaining what the `@param descriptor` refers to? Is there somewhere I reference to copy comments from?